scripts/globals/effects/mounted.lua

```diff
@@ -6,11 +6,9 @@ require("scripts/globals/status")
 local effect_object = {}
 
 effect_object.onEffectGain = function(target, effect)
-    --[[
-        Retail sends a music change packet (packet ID 0x5F) in both cases.
-    ]]
+    -- Retail sends a music change packet (packet ID 0x5F) in both cases.
 
-    if effect:getPower() == 0 then
+    if effect:getPower() < 2 then
```

Comment:
I think I missed this part on stream, value 1 is listed as the raptor on the quest so whats the deal here?

Reply:
True, this is copy/paste from Ivaar's branch, I can test to see if this matters or not

Reply:
Yeah this seems a bit off, I think there is a little more capturing to do. We are off-by-one _somewhere_ down the line for chocobo and mount stuff. Everything works currently, but it will want cleanup in the future. I'll do it as part of chocobo raising 👍